README.md

```diff
@@ -12,9 +12,22 @@ __Other components pending__
 
 
 ## Development
-Before the setup install `jq`, `bazelisk`, `docker` and `kubectl`.
+Before the setup install `jq`, `golang`, `make`, `bazelisk`, `docker`, `helm`, and `kubectl`:
+
+On Linux, jq, golang, make and docker can be installed using the package manager.
+
+Links for installing the other software:
+* Bazelisk: https://github.com/bazelbuild/bazelisk/releases
+* Helm: https://helm.sh/docs/intro/install/
+* Kubectl: https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/
+
 A local Kubernetes cluster is also required. Docker Desktop is a good solution.
 
+Configure kubectl to manage the local Kubernetes cluster by placing the
+KubeConfig yaml into `$HOME/.kube/config` and do `chmod 600 $HOME/.kube/config`
```

Comment:
Is this really necessary? You should really know how to use kubectl prior to installing Peridot imo.

Reply:
@hbjydev

Hey Hayed, I only want to help to make getting this running easier, making it less of a hurdle.

It depends on how inviting you want it be. If you want to build a larger community with many contributors it may help if you make it easier for them to join. They will grow over time to be more experienced, and if you can make it so that even an otherwise experienced developer without Kubernetes experience (after all it's just a tool to run it, we are not developing Kubernetes itself here), can get Peridot running in a few hours or a day in one, go, the chance of him becoming a contributor is much higher than if e.g. he has to find out what one is assumed to know. This simple sentence it much faster to read an execute than to run into the error later when installing istio.

I personally would like to see if the state of Peridot, and if I would be successful, my peers would also join the show.

Or is Peridot only open-sourced pro-forma (for marketing, or because someone decided to "throw the source over the wall", like with some some other pseudo-open-source projects), and really to invite users and contributions to grow the community around Rocky? I'm hoping that what this repo is about is to improve it, and that's that this PR wants.